tests/sample/data_usage/countbytes.c

```diff
@@ -0,0 +1,26 @@
+#include <stdbool.h>
+#include <stdint.h>
+#include "ebpf_helpers.h"
+#include "ebpf_nethooks.h"
+
+#pragma clang section data = "maps"
+ebpf_map_definition_t byte_map = {
+    .size = sizeof(ebpf_map_definition_t),
+    .type = EBPF_MAP_TYPE_ARRAY,
+    .key_size = sizeof(five_tuple_t),
+    .value_size = sizeof(uint64_t),
+    .max_entries = 500};
+
+
+#pragma clang section text = "mac"
+int
+CountBytes(mac_md_t* ctx)
+{
```

Comment:
Does this compile at all? You're missing the return

Reply:
Woops, I took out Done and forgot to add in the return. Thanks Eric